homeassistant/helpers/entity.py

```diff
@@ -551,6 +554,19 @@ def __repr__(self) -> str:
         """Return the representation."""
         return "<Entity {}: {}>".format(self.name, self.state)
 
+    # call an requests
+    async def async_request_call(self, request, params):
```

Comment:
should be `async_request_call(self, coro, *args, **kwargs)`

Reply:
I'm a bit stuck in the, and what are the changes to the rest of the code path, the caller, and the function call below. Could you help me with that
the whole `*` `**` magic in python is still a bit magic for me

Reply:
I hope 1d170e6 is indeed what you asked

Reply:
Yep, perfect